session_check.php

```diff
@@ -0,0 +1,42 @@
+<?php
+	/*
+	 * @author: mfsi-krishnadev
+	 * @class: checking session of logged-in user
+	*/
+
+	class session_check
+	{
+
+		/*
+		 * @description: checking session whether user is valid or not
+		 * @access: public
+		 * @param : none
+		 * return: boolean
+		*/
+		function logged_in()
+		{
+			//checking the session variable user_id is present or not
+			if(isset($_SESSION["user_id"]) && !empty($_SESSION["user_id"]))
+			{
+				return true;
+			}
+			else
```

Comment:
This else part is not required here.
directly can do
//checking the session variable user_id is present or not
if(isset($_SESSION["user_id"]) && !empty($_SESSION["user_id"]))
{
    return true;
}

return false;

----
because when if condition will get true, it will return true. if condition fails then it will execute rest of the code out present after else.